ntbs-service/DataMigration/ImportValidator.cs

```diff
@@ -89,48 +128,99 @@ public ImportValidator(IImportLogger logger, IReferenceDataRepository referenceD
             {
                 notification.TestData.HasTestCarriedOut = false;
             }
+        }
+
+        private async Task CleanContactTracingData(Notification notification, PerformContext context, int runId)
+        {
+            if (!ValidateObject(notification.ContactTracing).Any())
+            {
+                // If there are no issues in the contact tracing
+                return;
+            }
+
+            var messages = new List<string>();
+            var lostData = new List<string>();
+
+            // The order we clean properties is important, because if we overwrite one value (eg. AdultsScreened) with
+            // null then we also need to overwrite the values that are necessarily lower than it (eg. AdultsActiveTB)
+
+            CleanValidationProperty(nameof(ContactTracing.AdultsScreened), notification, messages, lostData);
+            CleanValidationProperty(nameof(ContactTracing.ChildrenScreened), notification, messages, lostData);
+
+            // Because Active + Latent <= Screened, these have to be cleaned together
+            CleanValidationPropertyPair(nameof(ContactTracing.AdultsActiveTB), nameof(ContactTracing.AdultsLatentTB),
+                notification, messages, lostData);
+            CleanValidationPropertyPair(nameof(ContactTracing.ChildrenActiveTB),
+                nameof(ContactTracing.ChildrenLatentTB), notification, messages, lostData);
+
+            var propertyNames = new List<string>
+            {
+                nameof(ContactTracing.AdultsStartedTreatment), nameof(ContactTracing.ChildrenStartedTreatment),
+                nameof(ContactTracing.AdultsFinishedTreatment), nameof(ContactTracing.ChildrenFinishedTreatment),
```

Comment:
Now that we're not treating all the fields on the same footing, I think it would be a good idea to get rid of this list entirely and just have a separate `CleanValidationProperty` for each one

Reply:
Yeah definitely, done